test/backend/CMakeLists.txt

```diff
@@ -79,6 +79,7 @@ else()
 endif()
 
 set(LD_PRELOADS $<TARGET_FILE:cruntime>)
+set(LD_LIBRARY_PATH /workdir/onnx-mlir/build/Debug/lib)
```

Comment:
The problem with this code is that "/workdir/onnx-mlir/build/Debug/lib" is valid only in a particular build environment. This is because a user might build the compiler in a different location (directory) than `/workdir/onnx-mlir`. Also we might do a `Release` build rather than a `Debug` build.  We will need to use a CMAKE variable to factor out those build environment specific locations.